### PR TITLE
Online tools and try sites

### PR DESCRIPTION
**Pull request by catinthetap**
Hi Charlotte - this isn't an exhaustive list but hopefully useful. Let me know what you think (just hope I haven't messaged up the PR!)
Jim

### CODE REVIEW

File: no-install/try-before-you-buy.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Starting coding without needing to install anything
+
+* [CodeSandbox](https://codesandbox.io/) - supports working in React, Angular, Vue, Preact and vanilla JS
```
Comment (charlottebrf) [inline no-install/try-before-you-buy.md]: Nice addition! Thanks 👍

File: README.md [hunk 2]
```diff
@@ -28,6 +28,7 @@ The current information I've collated includes:
 * [Learn To Code Resources](online-resources/learn-to-code-resources.md)
 * [Podcasts](online-resources/podcasts.md)
 * [Tests/Interviews/CV](online-resources/tech-tests-and-interviews.md)
+* [Getting started without installing](no-install/try-before-you-buy.md)
```
Comment (charlottebrf) [inline README.md]: Brilliant thanks for adding this! 💃

File: no-install/try-before-you-buy.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
 # Starting coding without needing to install anything
 
+In a lot of cases you can try working in a language (or with a technology) without committing the time to setting up tools on a computer (a process even very experience developers can find both time consuming and frustrating)
```
Comment (charlottebrf) [inline no-install/try-before-you-buy.md]: This is a really great addition. Thanks @catinthetap 👍

**Comment by catinthetap** [comment]
Glad it’s ok. Will try and tidy up the formatting too

**Pull request merged**